web/tests/compose_actions.test.js

```diff
@@ -462,6 +468,20 @@ test("quote_message", ({disallow, override, override_rewire}) => {
     });
     assert.ok(replaced);
 
+    opts.forward_message = true;
+    replaced = false;
+
+    override(compose_ui, "insert_and_scroll_into_view", noop);
+
+    quote_message(opts);
+
+    success_function({
+        raw_content: "Testing.",
+    });
+    assert.ok(replaced);
+
+    delete opts.forward_message;
```

Comment:
This test might be more readable if converted to just have `opts` be defined using `let` and totally new values selected for each little section of test. The serial mutations make it a lot harder to know what value it has at any given point.

Reply:
Updated this according to your suggestion